src/components/draggable/draggable.stories.mdx

```diff
@@ -85,7 +86,29 @@ import {
 ### With getOrder callback
 
 <Canvas>
-  <Story name="with getOrder callback" id="draggable-test--default" />
+  <Story name="with getOrder callback">
+    {() => {
+      const handleUpdate = (items) => {
+        action("onUpdate")(items);
+      };
+      return (
+        <DraggableContainer getOrder={handleUpdate}>
+          <DraggableItem key="1" id={1}>
+            <Checkbox label="Draggable Label One" />
+          </DraggableItem>
+          <DraggableItem key="2" id={2}>
+            <Checkbox label="Draggable Label Two" />
+          </DraggableItem>
+          <DraggableItem key="3" id={3}>
+            <Checkbox label="Draggable Label Three" />
+          </DraggableItem>
+          <DraggableItem key="4" id={4}>
+            <Checkbox label="Draggable Label Four" />
+          </DraggableItem>
+        </DraggableContainer>
+      );
+    }}
+  </Story>
 </Canvas>
 
 ## Props
```

Comment:
Not something for this PR (unless you really, really want to 🤷🏻 ) but the prop table for `Draggable` isn't the best. It doesn't state what `id` and `children` actually accept. I'm happy to raise another ticket separately for this, just let me know 😄 
![Screenshot 2022-01-21 at 11 30 02](https://user-images.githubusercontent.com/56251247/150519842-aeb4335c-3ebc-40ed-8e24-8321120f8494.png)

Reply:
Created FE-4868 for this issue.